### DISCUSSION TITLE
[Feature Request] select multi individual items

### DISCUSSION THREAD

**Discussion by caghetti** [josdejong/svelte-jsoneditor#356] [Ideas]
Today it's possible to select several items via a shift+clic. 
A nice feature would be to have the possibility to select multiple individual items

like this

![image](https://github.com/josdejong/svelte-jsoneditor/assets/148205032/dec096a4-bbda-412b-9d07-f3fb272b9870)

**Comment by caghetti** [comment]
Hi @josdejong. I ping you about this feature request : do you think you'll implement it ?

**Comment by josdejong** [comment]
Hi @caghetti , I like the idea but I have no plans to implement this anytime soon myself. Since this is an open source project, anyone can help implementing this if there is need for it.

Since we're not actively working on this idea right now, I'll move it to the "Discussions | Ideas" section.

**Reply by caghetti** [reply]
Hi, I understand.
We'll check with my team if we can to a PR on the subject (it will be in january / february). If we need help about the structure of your code, can we ping you ?
